category.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace PHPMD\Category;
+
+class Category
+{
+    public static $checks;
+}
+
+Category::$checks = array(
```

Comment:
Maybe assigning this *within* the class definition, instead of after (as a side-effect of interpreting the file) will avoid the CC issue?